src/react/components/Link.tsx

```diff
@@ -47,6 +51,10 @@ export const Link = <Match,>({
   );
 };
 
+export const Link = forwardRef(RawLink) as <Match>(
+  p: LinkPropsWithRef<Match>
+) => ReactElement | null;
+
```

Comment:
casting component type, since Link Component is generic component, forwardRef function can't determine the Component type.
[reference link](https://stackoverflow.com/questions/58469229/react-with-typescript-generics-while-using-react-forwardref/58473012)